### DISCUSSION TITLE
Canvas

### DISCUSSION THREAD

**Discussion by b18050** [sugarlabs/musicblocks-v4#54] [General]
Canvas MVC structure 
- Graphics 
- canvas
*Model*
- data (states)
*View*
-
*Controller*
- interaction point for model and view. Also the connection point for monitor.

**Comment by meganindya** [comment]
Try to list the variables and methods each for the model and the view. Also, list the methods one needs to communicate with the other, and the methods the controller needs to communicate (bidirectionally) with the monitor.

**Comment by meganindya** [comment]
Create an issue like the other have, when you have the contents :D
